Documentation/tutorials/map-track-data-using-data-prep.md

```diff
@@ -0,0 +1,164 @@
+Data Prep is an Adobe Experience Platform service which maps and transforms data to the [Experience Data Model (XDM)](https://experienceleague.adobe.com/docs/experience-platform/xdm/home.html).  Data Prep is configured from a Platform enabled [datastream](https://experienceleague.adobe.com/docs/experience-platform/edge/datastreams/overview.html) to map source data from the Edge Bridge mobile extension to the Platform Edge Network.
+
+This guide covers how to map data sent from the Edge Bridge within the Data Collection UI.
+
+For a quick overview of the capabilities of Data Prep, watch the following [video](https://experienceleague.adobe.com/docs/platform-learn/data-collection/edge-network/data-prep.html).
+
+> **Note**
+> The following documentation provides a comprehensive overview of the Data Prep capabilities:
+> - [Data Prep overview](https://experienceleague.adobe.com/docs/experience-platform/data-prep/home.html)
+> - [Data Prep mapping functions](https://experienceleague.adobe.com/docs/experience-platform/data-prep/functions.html)
+> - [Handling data formats with Data Prep](https://experienceleague.adobe.com/docs/experience-platform/data-prep/data-handling.html)
+>
+
+## Prerequisites
+Before continuing, create an XDM schema and configure a datastream using the following tutorials. In order to send data to the Edge Network, the datastream must be configured with the Adobe Experience Platform service.
+- [Create a schema](https://experienceleague.adobe.com/docs/platform-learn/implement-mobile-sdk/initial-configuration/create-schema.html)
+- [Createa  datastream](https://experienceleague.adobe.com/docs/platform-learn/implement-mobile-sdk/initial-configuration/create-datastream.html)
+
+In the following steps, the examples assume the schema configured with the datastream contain these field groups:
+- AEP Mobile Lifecycle Details
+- Adobe Experience Edge Autofilled Environment Details
+- Commerce Details
+
+## Select data
+While creating a new datastream, select **Save and Add Mapping** to go to the **Select data** step directly.  Alternately, if using an already saved datastream, select **Edit Mapping**. From here, provide a sample JSON object that represents the data layer used in the application as sent from the Edge Bridge mobile extension.
+
+The following are examples of the JSON objects sent from the Edge Bridge mobile extension. The Experience event sent from the Edge Bridge extension contains both **xdm** and **data** JSON objects. The **data** object contains the context data passed to the `MobileCore.trackAction()` and `MobileCore.trackState()` APIs.
+
+The following is an example of an `MobileCore.trackAction` API call and the resulting JSON Experience event sent to the Edge Network. Note the JSON is truncated to show just the relevant data.
+```swift
+MobileCore.track(action: "add_to_cart", data: ["product.id": "12345", "product.add.event": "1", "product.name": "wide_brim_sunhat", "product.units": "1"])
+```
```

Comment:
This code-snippet needs to be converted to java as well (you can use the test app examples)

Reply:
Updated snippet for java and kotlin examples